Makefile

```diff
@@ -19,7 +19,7 @@ install:
 	./hide_tbd_warning go install ./...
 
 install-dev:
-	if ! [[ -e "$(SYNCLET_DEV_IMAGE_TAG_FILE)" ]]; then echo "No dev synclet found. Run make synclet-dev."; exit 1; fi
+	@if ! [[ -e "$(SYNCLET_DEV_IMAGE_TAG_FILE)" ]]; then echo "No dev synclet found. Run make synclet-dev."; exit 1; fi
```

Comment:
`@` so that the command isn't printed so that people don't spend time wondering about the error message on the screen when there's not actually an error

Reply:
Ah, thanks!

Reply:
(This uh. Seems like poor design on the part of Make...)